crates/forge/src/runner.rs

```diff
@@ -335,14 +352,24 @@ impl<'a> ContractRunner<'a> {
                     ..
                 }) => {
                     traces.extend(execution_trace.map(|traces| (TraceKind::Execution, traces)));
-                    labeled_addresses.extend(new_labels);
+                    labeled_addresses.extend(
+                        new_labels
+                            .into_iter()
+                            .map(|l| (b160_to_h160(l.0), l.1))
+                            .collect::<Vec<_>>(),
```

Comment:
no need to collect

Reply:
done in https://github.com/foundry-rs/foundry/pull/5768/commits/3e18639d0ca47edd9c4d8aed36e8c14300eed8fa